Mathlib/Topology/Order/DenselyOrdered.lean

```diff
@@ -229,6 +229,33 @@ theorem comap_coe_nhdsWithin_Iio_of_Ioo_subset (hb : s ⊆ Iio b)
     obtain ⟨x : s, hx : ∀ z, x ≤ z → z ∈ u⟩ := mem_atTop_sets.1 hu
     exact ⟨Ioo x b, Ioo_mem_nhdsWithin_Iio' (hb x.2), fun z hz => hx _ hz.1.le⟩
 
+/-- A set with a point in both the frontier and the set is not open -/
+private lemma frontier_to_not_IsOpen{α: Type*} [TopologicalSpace α] {s: Set α} {a: α}
```

Comment:
I see no reason for this to be private. You can rename it as `not_isOpen_of_mem_of_mem_frontier`.